Samples/Notifications/Push/cpp-console-unpackaged/cpp-console-unpackaged.cpp

```diff
@@ -0,0 +1,203 @@
+
+#include <iostream>
+
+#include <wil/result.h>
+#include <wil/cppwinrt.h>
+
+#include <winrt/Windows.ApplicationModel.Background.h>
+#include <winrt/Windows.Foundation.h>
+#include <winrt/Microsoft.Windows.AppLifecycle.h>
+#include <winrt/Microsoft.Windows.PushNotifications.h>
+#include <winrt/Windows.Globalization.DateTimeFormatting.h>
+
+#include <MddBootstrap.h>
+
+using namespace winrt::Microsoft::Windows::AppLifecycle;
+using namespace winrt::Microsoft::Windows::PushNotifications;
+using namespace winrt::Windows::Foundation;
+using namespace winrt::Windows::Globalization::DateTimeFormatting;
+
+// To obtain an AAD RemoteIdentifier for your app,
+// follow the instructions on https://docs.microsoft.com/en-us/windows/apps/windows-app-sdk/notifications/push/push-quickstart#configure-your-apps-identity-in-azure-active-directory
+winrt::guid remoteId{ "00000000-0000-0000-0000-000000000000"}; // Replace this with your own RemoteId
+
+HRESULT LoadWindowsAppSDK()
+{
+    // Major.Minor version, MinVersion=0 to find any framework package for this major.minor version
+    const UINT32 majorMinorVersion{ 0x00010001 }; //  { Major << 16) | Minor };
+    PCWSTR versionTag{ L"" };
+    const PACKAGE_VERSION minVersion{};
+    HRESULT hr{ MddBootstrapInitialize(majorMinorVersion, versionTag, minVersion) };
+    if (FAILED(hr))
+    {
+        wprintf(L"\nError 0x%08X in MddBootstrapInitialize(0x%08X, %s, %hu.%hu.%hu.%hu)\n",
+            hr, majorMinorVersion, versionTag, minVersion.Major, minVersion.Minor, minVersion.Build, minVersion.Revision);
+        return hr;
+    }
+
+    return S_OK;
+}
+
+winrt::Windows::Foundation::IAsyncOperation<PushNotificationChannel> RequestChannelAsync()
+{
+    auto channelOperation = PushNotificationManager::Default().CreateChannelAsync(remoteId);
+
+    // Setup the in-progress event handler
+    channelOperation.Progress(
+        [](auto&& sender, auto&& args)
+        {
+            if (args.status == PushNotificationChannelStatus::InProgress)
+            {
+                // This is basically a noop since it isn't really an error state
+                std::cout << "\nWNS ChannelURI request is in progress." << std::endl;
+            }
+            else if (args.status == PushNotificationChannelStatus::InProgressRetry)
+            {
+                LOG_HR_MSG(
+                    args.extendedError,
+                    "The WNS ChannelURI request is in back-off retry mode because of a retryable error! Expect delays in acquiring it. RetryCount = %d",
+                    args.retryCount);
+            }
+        });
+
+    auto result = co_await channelOperation;
+
+    if (result.Status() == PushNotificationChannelStatus::CompletedSuccess)
+    {
+        auto channel = result.Channel();
+
+        DateTimeFormatter formater = DateTimeFormatter(L"on {month.abbreviated} {day.integer(1)}, {year.full} at {hour.integer(1)}:{minute.integer(2)}:{second.integer(2)}");
```

Comment:
Can you comment an example of the output above this?

Reply:
Why? this is self explanatory and whenever someone wants to tweak the format, they'll have to update the comment.

Reply:
Most of it is. But what does day.integer(1) mean? How is it different that minute.integer(2)? Can't a day be 2 digits? I'm not familiar with DateTimeFormatter but it would be helpful to have a sample out to understand.